squad/frontend/templates/squad/_metadata.html

```diff
@@ -0,0 +1,12 @@
+<div class='metadata'>
+    {% for key, value in metadata %}
+    <div class='row'>
+        <div class='col-md-3 col-sm-3'>
+            <strong>{{key}}</strong>
+        </div>
+        <div class='col-md-9 col-sm-9'>
+            {{value|urlize}}
```

Comment:
In case of lists (a few commits later) I think each list item should be displayed in separate line. Is there an easy way to do that or we'll need to create a templatetag?